streaming/index.js

```diff
@@ -195,26 +207,33 @@ const startServer = async () => {
 
   app.set('trust proxy', process.env.TRUSTED_PROXY_IP ? process.env.TRUSTED_PROXY_IP.split(/(?:\s*,\s*|\s+)/) : 'loopback,uniquelocal');
 
+  app.use(httpLogger);
   app.use(cors());
 
   // Handle eventsource & other http requests:
   server.on('request', app);
 
   // Handle upgrade requests:
   server.on('upgrade', async function handleUpgrade(request, socket, head) {
+    const startTime = Date.now();
+
```

Comment:
This is only used for the responseTime logging, which I'm not sure if we really need here, so we could remove this and line 267.

Reply:
I don't feel strongly either way. I think this is not useful in most of cases, but it could be in some?